test/test/missing_required_interfaces.cpp

```diff
@@ -0,0 +1,56 @@
+#include "pch.h"
+
+// Unset lean and mean so we can implement a type from the test_component namespace
+#undef WINRT_LEAN_AND_MEAN
+#include <winrt/test_component.h>
+
+namespace
+{
+    struct LiesAboutInheritance : public winrt::implements<LiesAboutInheritance, winrt::test_component::ILiesAboutInheritance>
+    {
+        LiesAboutInheritance() = default;
+        void StubMethod() {}
+    };
+}
+
+bool g_failfastCalled = false;
+
+void __stdcall WINRT_IMPL_RoFailFastWithErrorContext(int32_t) noexcept
+{
+    g_failfastCalled = true;
+}
+
+void DoTheUncatcheable(winrt::test_component::LiesAboutInheritance& lies) noexcept
```

Comment:
nit: spelling `uncatchable` in both methods

Reply:
Now that this throws instead of failfasting that stuff is deleted, typo and all.